app/errors.js

```diff
@@ -21,3 +19,8 @@ exports.defaultError = message => {
     message
   };
 };
+
+exports.invalidUser = {
+  statusCode: 422,
```

Comment:
When someone puts invalid credentials we usually respond with 401 or 403 status code.